### ISSUE TITLE
Remove Google Recaptcha

### ISSUE THREAD

**Issue by dansup** [pixelfed/pixelfed#1140]
Google Recaptcha was added as an optional feature for login form and registration form. It does not meet our standards for accessibility and privacy and needs to be removed.

### Todo
- [ ] Remove from ```LoginController```
- [ ] Remove from ```RegisterController```
- [ ] Remove from Checkpoint & Sudo Mode
- [ ] Remove from ```login.blade.php```
- [ ] Remove from ```register.blade.php```
- [ ] Remove from ```config/pixelfed.php```
- [ ] Remove from ```composer.json```
- [ ] Remove from docs

**Comment by lapineige** [comment]
What about implement (still as an opt-in option) a Captcha that contribute to the commons ?
I've heard of this one https://wiki.hsr.ch/StefanKeller/ReMAPTCHA, but I did not check if it was realy ethical or if some better alternatives are available.

**Comment by ealgase** [comment]
I think the CoinHive captcha might also be a good choice.

**Comment by shleeable** [comment]
> I think the CoinHive captcha might also be a good choice.

mmmm climate change damaging bitcoin is unethical... no coinhive please

**Comment by lapineige** [comment]
I agree.
If we force humans (and robots) to execute some task (as a captcha), it's better if it contributes to the commons instead of purely wasting that energy.

**Comment by tadzik** [comment]
Coinhive is unfortunately dead (https://coinhive.com/blog/en/discontinuation-of-coinhive), and they didn't answer my questions about opensourcing their captcha code, so unfortunately that's a dead end :(

As for the climate damage claims, I'd be very much interested in seeing how [hydropowered facilities utilizing energy that would otherwise go to waste](https://medium.com/coinshares/beware-of-lazy-research-c828c900b7d5) are damaging the climate, but I don't think that discussion is the point of this issue, and so it may be better off continued elsewhere :)

**Comment by MarkMura** [comment]
So many people falling for the "Bitcoin causes climate change" hoax...
I wasn't a big fan of Coinhive either, but for different reasons. Unfortunately ReMaptcha is completely unworkable, since it was just a research project in 2014. I'd love to see something similar come to life, but ReMaptcha is simply not it.

**Comment by ealgase** [comment]
It's not about the cryptocurrency produced (there are ethical concerns about crypto), it's about ensuring a way that's difficult to automate.  Since you'd need to spend significant CPU power to create spam accounts, it wouldn't be worth it.

**Comment by trwnh** [comment]
Fixed by #1231
